src/helpers/visibilityClassnames.ts

```diff
@@ -17,6 +17,8 @@ export const displayFrom = (from: Size) => {
       return display('hidden', 'sm:flex')
     case 'md':
       return display('hidden', 'md:flex')
+    case 'body':
+      return display('hidden', 'body:flex')
```

Comment:
What's the new size `body` for?

Reply:
It's body container, when the screen edges will touch this red square the styles will change (so it's both centre card width and breakpoint for screens)
![image](https://user-images.githubusercontent.com/49658988/191579094-f918e70b-06d5-46f9-a3bc-c2fae6923bb2.png)